### DISCUSSION TITLE
Feature request: ability to refuse sponsorships from specific countries (Finland)

### DISCUSSION THREAD

**Discussion by Mikaela** [community/community#14855] [Other Feature Feedback, Questions, & Ideas]
Finland has a strict money gathering law and only associations can apply for a permit to collect money. I think GitHub Sponsors counts as an international crowdfunding platform and thus by my interpretation GitHub Sponsors may be illegal unless it's possible to opt-out of getting donations from Finnish residents:

> **We intend to use an international crowdfunding platform. Do we still need a money collection permit?**
A money collection permit is always required if your donors are in Finland. Your donors are deemed to be in Finland if you are appealing to the Finnish public.
> 
> Simply using an international website or making your appeal in a language other than Finnish or Swedish does not mean that you can run your campaign without a money collection permit or without filing a small-scale money collection notice. However, if you are clearly limiting your appeal to an audience outside of Finland and your campaign is deemed not to be taking place in Finland, then no money collection permit is required.

* https://poliisi.fi/en/crowdfunding-and-money-collection-campaigns

Judging by an earlier section of the page, I understand in eyes of Finnish legislation the sponsors-only repositories wouldn't be enough either.

> All parties involved in the transaction must understand that they are engaging in trading: the buyers must know that they are buying a product and not just donating money or supporting a cause, and the seller must recognise that they are selling products rather than asking for donations.

*I am not a lawyer.*

**Comment by jlord** [comment]
Hi! Thanks for this feedback, it's really helpful, we're exploring how to better support different regions ✨

**Reply by medavidallsop** [reply]
I would like this feature, any progress since this was posted a couple of years ago?

**Reply by Larhzu** [reply]
Based on the discussion I had with the Finnish authorities, I understood that an ability to block donations from Finland would be useful. Then it's clear that one isn't soliciting money from people living in Finland, and thus the money collection act doesn't apply. So it would be nice if GitHub Sponsors had such a feature.

**Comment by seangemini2025** [comment]
bump
